internal/cli/federatedauthentication/identityprovider/identity_provider.go

```diff
@@ -30,6 +30,7 @@ func Builder() *cobra.Command {
 	cmd.AddCommand(
 		DeleteBuilder(),
 		DescribeBuilder(),
+		ListBuilder(),
```

Comment:
for later but this will print int he order you define them, starting with delete is odd, usually start with create or list